src/chat/chatter.py

```diff
@@ -271,20 +274,26 @@ def updatePlayer(self):
     def update_country(self):
         player = self.user_player
         if player is None:
+            if self.toolTip() == "":  # EARLY OUT
```

Comment:
Why is it necessary here? Also doing logic based on the state of the UI is bad practice.